cmd.c

```diff
@@ -897,6 +882,75 @@ void cmd_syspage(char *s)
 }
 
 
+void cmd_phfs(char *s)
+{
+	u16 argsc = 0;
+	unsigned int i, major, minor, pos = 0;
+	char args[MAX_CMD_ARGS_NB][LINESZ + 1];
+
+	for (i = 0; argsc < MAX_CMD_ARGS_NB; ++i) {
+		cmd_skipblanks(s, &pos, ". \t");
```

Comment:
The `cmd_skipblanks()` function is called at the begining of the `cmd_getnext()` function.

Reply:
You have right. Fixed